infrastructure/router/router.go

```diff
@@ -0,0 +1,31 @@
+package router
+
+import (
+	"WizelineApi/interface/controller"
+	//"academy-go-q42021/interface/controller"
+	"log"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+)
+
+
+
+func NewRouter() {
+	r := mux.NewRouter()
+
+	handler_pkm := controller.NewPokemonController()
```

Comment:
Please avoid the use of `snake_case`. Go's recommendation is to use `camelCase` and `PascalCase` only.

Also, IMHO, it's better to name it `pokemonHandler`, it has better readability, and also there's no need to save some space by shortening the `pokemon` word.

```suggestion
	pokemonHandler := controller.NewPokemonController()
```